### DISCUSSION TITLE
Change icon of the burn bundle using custom bootstrapper wixBA

### DISCUSSION THREAD

**Discussion by sarojnayak1991** [wixtoolset/issues#8787] [WiX Development]
Hello,

Is there any way to change the control panel icon of the wix bundle exe using custom bootstrapper WixBA.I could see we can set the icon of the burn bundle exe using bundle parameter **IconSourceFile** which is fixed.I have a requirement of changing this icon dynamically based on certain registry condition.

Regards,
Saroj

**Comment by robmen** [comment]
> I have a requirement of changing this icon dynamically

That's not supported today. I've never seen that requirement before. Learned something new today.
